lib/ansible/modules/identity/cyberark/cyberark_user.py

```diff
@@ -0,0 +1,461 @@
+#!/usr/bin/python
+# Copyright: (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: cyberark_user
+short_description: Module for CyberArk User Management using PAS Web Services SDK
+author: Edward Nunez @ CyberArk BizDev (@enunez-cyberark, @cyberark-bizdev, @erasmix)
+version_added: 2.4
+description:
+    - CyberArk User Management using PAS Web Services SDK. It currently supports the following
+      actions Get User Details, Add User, Update User, Delete User.
+
+
+options:
+    username:
+        required: True
+        description:
+            - The name of the user who will be queried (for details), added, updated or deleted.
+    state:
+        default: details
```

Comment:
The default should either be one of the choices or None if this is a tristate (ie: not being explicitly set by the user has a special meaning).